code/main.js

```diff
@@ -1,7 +1,10 @@
-console.log("starting app");
-let counter = 0
-setInterval(() => {
-    console.log(counter+=1);
-}, 1000);
+import express from 'express';
 
+const PORT = 3000;
 
+const app = express();
+
+// Middleware for static directory.
```

Comment:
Avoids an extra network request and has the added benefit of now showing two extra slugs '/static/index.html' in url.